apps/api_web/lib/api_web/controllers/service_controller.ex

```diff
@@ -99,6 +99,50 @@ defmodule ApiWeb.ServiceController do
               format: :date,
               example: "2018-12-24"
             )
+
+            service_description(
+              :string,
+              "Human-readable description of the service, as it should appear on public-facing websites and applications.",
+              "x-nullable": true,
+              example: "Weekday schedule (no school)"
+            )
+
+            service_schedule_name(
+              :string,
+              "Description of when the `service_id` is in effect.",
+              "x-nullable": true,
+              example: "Weekday (no school)"
+            )
+
+            service_schedule_type(
+              :string,
+              """
+              Description of the schedule type the service_id can be applied.
+              For example, on a holiday, the service_schedule_type value may be "Saturday" or "Sunday".
+              """,
+              "x-nullable": true,
+              enum: [nil, "Weekday", "Saturday", "Sunday", "Other"],
+              example: "Sunday"
+            )
+
+            service_schedule_typicality(
+              :integer,
+              """
+              Describes how well this schedule represents typical service for the listed `service_schedule_type`
+
+              | Value | Description                                                                 |
+              |-------|-----------------------------------------------------------------------------|
+              | `0`   | (or null) Not defined.                                                      |
```

Comment:
In the API, should never be `null`: we should convert it to `0` at parsing.

Reply:
If there is no calendar_attributes entry for a service, so the other fields are all null, should we still fill this with 0?

Reply:
Yep; it makes it much easier for clients.